build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -418,11 +279,23 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
             }
 
             // cleanup compilers project output we don't need
-            DeleteMainPublishOutput(outputDir, "compilers");
+            PublishMutationUtilties.CleanPublishOutput(outputDir, "compilers");
             File.Delete(compilersDeps);
-            File.Delete(compilersRuntimeConfig);
+            
+            // Publish SharedFx
+            var sharedFrameworkNugetVersion = c.BuildContext.Get<string>("SharedFrameworkNugetVersion");
+            var commitHash = c.BuildContext.Get<string>("CommitHash");
+
+            var sharedFrameworkPublisher = new SharedFrameworkPublisher(
```

Comment:
They are used for separate things, dotnet-host-build publishes it to a different directory to create zip packages, dotnet-cli-build publishes it to the stage directories to create appropriately constructed, usable layouts.